sql/src/main/java/io/crate/operation/RowFilter.java

```diff
@@ -63,13 +61,17 @@ public boolean apply(@Nullable Row row) {
     public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
+
         RowFilter rowFilter = (RowFilter) o;
-        return Objects.equal(filterCondition, rowFilter.filterCondition) &&
-               Objects.equal(filterCollectExpressions, rowFilter.filterCollectExpressions);
+
+        if (!filterCondition.equals(rowFilter.filterCondition)) return false;
```

Comment:
Can you please add braces to this if statement (lack of braces can be a source of bugs) ?